education/dina_independent/dina_independent.Rmd

```diff
@@ -449,4 +448,4 @@ ggplot(data=estimates, aes(x=mle, y=post.means, shape=pars)) + geom_point()  + g
 
 # References
 
-<!-- This comment causes section to be numbered -->
\ No newline at end of file
+<!-- This comment causes section to be numbered -->
```

Comment:
[not required for this PR]

Are these lines different?  You can also just specify whether sections are numbered in the top-level yaml.

Reply:
Hi @bob-carpenter ! Thanks for reviewing.
I didn't mean to submit this change specifically; it's the text editor (in this case, GitHub's editor, but Vim would do the same thing) which adds a "newline at end of file" whenever it's missing.
Indeed, that's what the command-line diff looks like:
```
-<!-- This comment causes section to be numbered -->
\ No newline at end of file
+<!-- This comment causes section to be numbered -->
```
I would think it's a healthy change although very minor.